packages/idafree.vm/idafree.vm.nuspec

```
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>idafree.vm</id>
-    <version>8.3</version>
+    <version>8.3.20231128</version>
```

```suggestion
    <version>8.3.0.20231128</version>
```

We only use the forth segment for the date (not the third one): https://github.com/mandiant/VM-Packages/wiki/Package-Structure#non-metapackages